azure-functions/acbs-function/constants/deal.js

```diff
@@ -24,10 +24,24 @@ const CURRENCY = {
   DEFAULT: 'GBP',
 };
 
+const COUNTRY = {
+  DEFAULT: 'GBR',
+};
+
+const UNITED_KINGDOM = [
+  'united kingdom',
```

Comment:
are they supposed to be lowercased?

Reply:
I have converted it to lower case for comparison, I also convert the country to lower case for true like comparison.